js/app.js

```diff
@@ -12,4 +15,40 @@ export default {
   getSocketUrl() {
     return 'ws://localhost:8080/ws';
   },
+
+  followUnfollow(guid, type = 'follow') {
```

Comment:
I'm moving it to a followUtils.js file in utils, I think there could be other functions that can go in there that will be useful.